### ISSUE TITLE
Consolidate functionality of Libs/Alignment, including removing copied itk files

### ISSUE THREAD

**Issue by cchriste** [SCIInstitute/ShapeWorks#429]
This will be necessary to complete the consolidation of commands currently in Libs/Alignment:
- ResizeOriginResampleVolumes
- ResampleVolumeToIsotropic
- TranslationTransform
- ICPRigid3DImageRegistration
- ApplyRigid3DTransformationToImage
- ICPRigid3DMeshRegistration
- ReflectMeshes
- TranslateShapeToImageOrigin
- TranslateImages

The itk classes in Libs/Alignment/Transforms have been copied from an old version of ITK, and some may have been modified. The best way to accomplish their removal might be to modify the existing commands so they are no longer necessary, or it might be to focus on the consolidation, since these commands may be duplicates of other work as has been seen many times in this process. 

Regardless, this is going to require cooperation from someone who understands what these modified ITK classes are intended to accomplish. It could be like our friend, `DeepCopy`, which was created in the first place due to a misunderstanding of the ITK pipelines.

*First*, we should identify which of these commands are used with our current Python examples (adding @AtefehKashani and @jadie1 for this), and which existing shell scripts might make use of these (adding @prafulag and @riddhishb to assist). This will help us prioritize what to fix first.

Please keep in mind that the operations of these commands may not be appropriate to live in their own class and should instead be members of existing classes such as Image and Mesh.

This spreadsheet should be updated to reflect the timeline and relevance of the conversion of the underlying functionality of these commands to be part of the ShapeWorks library: https://docs.google.com/spreadsheets/d/17gevfSR-bY3cSr0C-rAmOe6kE8OtrFv06iWt_W10T1M/edit#gid=0

**Comment by cchriste** [comment]
Here is a list of all the commands from Alignment (found in Libs/Alignment/CMakeLists.txt) used in Examples/Python/*.py):
```bash
(shapeworks) cam@ananda:~/code/ShapeWorks/ShapeWorks/build_xcode$ find ../Examples/Python/ -name "*.py" -exec grep "ResizeOriginResampleVolumes\|ResampleVolumeToIsotropic\|TranslationTransform\|ICPRigid3DImageRegistration\|ApplyRigid3DTransformationToImage\|ICPRigid3DMeshRegistration\|ReflectMeshes\|TranslateShapeToImageOrigin\|TranslateImages" {} \; -print
            execCommand = ["TranslateShapeToImageOrigin", "--inFilename", innameSeg, "--outFilename", outnameSeg, "--useCenterOfMass",  "1", 
            execCommand = ["TranslateShapeToImageOrigin", "--inFilename", inname, "--outFilename", outname, "--useCenterOfMass", "1",
            execCommand = ["ICPRigid3DImageRegistration", "--targetDistanceMap", ref_tpdtnrrdfilename, "--sourceDistanceMap", tpdtnrrdfilename, "--sourceSegmentation", seginname, "--sourceRaw", rawinname, "--icpIterations", str(
            execCommand = ["ICPRigid3DImageRegistration", "--targetDistanceMap", ref_tpdtnrrdfilename, "--sourceDistanceMap", tpdtnrrdfilename, "--sourceSegmentation", inname, "--icpIterations", str(
../Examples/Python//GroomUtils.py
```
This already helps us narrow the scope of this task from nine commands down to two commands. Others involved should help identify which of the other commands should be consolidated later, if at all.

**Comment by archanasri** [comment]
Addressed in #905
